pkg/insights/insightsclient/insightsclient.go

```diff
@@ -33,7 +33,7 @@ import (
 const (
 	responseBodyLogLen = 1024
 	insightsReqId      = "x-rh-insights-request-id"
-	scaArchPayload     = `{"type": "sca","arch": "x86_64"}`
```

Comment:
Hey @Prashanth684 @aleskandro since we have multi-arch compute.... I'm assuming this would not work? Thoughts? Thanks, Paul

Reply:
just read the rest of the code... makes sense